source_account/setup_ec2.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
Could add `set -euxo pipefail` to make this safer. https://sipb.mit.edu/doc/safe-shell/

Reply:
I need to change to `set +e`. Since the cloud-init will exit as soon as there is an error during one step, but it will not throw any error. To make sure every step will be run, I need to ignore the error.